src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -113,8 +114,12 @@ class Filter extends Component<Props> {
         <Radio
           selected={filters.state[category] === count.id}
           value={count.id}
-          onSelect={v => {
-            return (filters as any).setFilter(category, v)
+          onSelect={selected => {
+            if (selected) {
+              return (filters as any).setFilter(category, count.id)
+            } else {
+              return (filters as any).unsetFilter(category)
```

Comment:
Might be good to add a `TODO` to later to correct whatever type issue is forcing you to coerce filters to any.

Reply:
Oh yea, duh- I can just type this higher up in the file!

I'm still finding these things in Typescript where I'm messing around and overlook an obvious solution 😭